ocs_ci/ocs/ripsaw.py

```diff
@@ -75,14 +76,14 @@ def _clone_ripsaw(self):
             log.error('Error during cloning of ripsaw repository')
             raise cf
 
-    def apply_crd(self, crd):
+    def apply_crd(self, crd, drop_cache):
         """
         Apply the CRD
 
         Args:
             crd (str): Name of file to apply
+            drop_cache (bool): Enable if OSD cache needs to be dropped
```

Comment:
where you are using this?
i do not see and deployment of this pod (drop_cache)

Reply:
I've removed this.